WordPress/Classes/ViewRelated/Menus/Views/Menu+ViewDesign.h

```diff
@@ -0,0 +1,12 @@
+
+extern CGFloat const MenusDesignStrokeWidth;
+extern CGFloat const MenusDesignDefaultCornerRadius;
+extern CGFloat const MenusDesignDefaultContentSpacing;
+
+#import "Menu.h"
```

Comment:
Even though `extern` declarations can go before the `imports`, I'd suggest moving the `import` directive to the top here, to maintain consistency with the rest of the codebase.

Nitpicky! 😄